.github/scripts/clone_oca_dependencies.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+# =============================================================================
+# Clone OCA Dependencies Script
+# =============================================================================
+#
+# This script clones OCA (Odoo Community Association) repositories listed in
+# the oca_dependencies.txt file. These dependencies are required for testing
+# and running Odoo addons that depend on modules from other OCA repositories.
+#
```

Comment:
Since the script can clone all repositories listed in `oca_dependencies.txt` even if they aren't from OCA, we can omit references to OCA whenever it isn't required.